platform-ui-skin/src/main/webapp/skin/less/core/components/Textarea/Style.less

```diff
@@ -20,7 +20,6 @@
 
 .extended-textarea {
   textarea {
-    min-width: 100%;
```

Comment:
why deleting this? you can use css helpers to force switching this value to other value knowing that CSS helpers uses `!important`

Reply:
ok

Reply:
ok

Reply:
ok

Reply:
Not needed!! I'll delete the PR